dvc/repo/get.py

```diff
@@ -53,26 +60,24 @@ def get(url, path, out=None, rev=None):
             #
             # Also, we can't use theoretical "move" link type here, because
             # the same cache file might be used a few times in a directory.
```

Comment:
This comment should stay over `repo.cache.local.cache_types = ["reflink", "hardlink", "copy"]` line, otherwise it looses some of its meaning.

Reply:
Oops